vizro-ai/changelog.d/20240220_142800_jo_stichbury_add_favicon.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Added
+
+- Added favicon to docs
```

Comment:
```suggestion
```
Again would remove ;) So this may seem annoying to create the files, but to not often fill them with content, but we found that it is a good reminder to create them and actively decide if this is CHANGELOG worthy or not